### PR TITLE
changed format to fstring literals

### PR DESCRIPTION
**Pull request by yukilikespie**

### CODE REVIEW

File: tor/helpers/flair.py [hunk 1]
```diff
@@ -138,8 +136,6 @@ def set_meta_flair_on_other_posts(config):
             post.link_flair_text != flair.meta
         ):
             logging.info(
-                'Flairing post {} by author {} with Meta.'.format(
-                    post.fullname, post.author
-                )
+                f'Flairing post {post.fullname} by author {post.author} with Meta.'
```
Comment (perryprog) [inline tor/helpers/flair.py]: This line is too long

File: tor/core/admin_commands.py [hunk 2]
```diff
@@ -40,11 +40,7 @@ def process_override(reply, config):
     parents_parent = config.r.comment(id=clean_id(reply_parent.parent_id))
     if 'done' in parents_parent.body.lower():
         logging.info(
-            'Starting validation override for post {}'
-            ', approved by {}'.format(
-                parents_parent.fullname,
-                reply.author.name
-            )
+            f'Starting validation override for post {parents_parent.fullname}, approved by {reply.author.name}'
```
Comment (perryprog) [inline tor/core/admin_commands.py]: This line is too long

File: tor/core/admin_commands.py [hunk 3]
```diff
@@ -101,25 +96,20 @@ def process_blacklist(reply, config):
         reply.reply(results)
 
         logging.info(
-            "Blacklist: {failed} failed, {success} succeeded, {ignored} were "
-            "already blacklisted".format(
-                failed=repr(failed),
-                success=repr(successes),
-                ignored=repr(already_added)
-            )
+            f"Blacklist: {repr(failed)} failed, {repr(successes)} succeeded, {repr(already_added)} were already blacklisted"
```
Comment (perryprog) [inline tor/core/admin_commands.py]: This line is too long

File: tor/core/inbox.py [hunk 4]
```diff
@@ -25,15 +25,10 @@
 
 def forward_to_slack(item, config):
     send_to_slack(
-        'Unhandled inbox reply by *{author}* -- *{subject}*: {body}'.format(
-            author=item.author,
-            body=item.body,
-            subject=item.subject,
-        ), config)
+        f'Unhandled inbox reply by *{item.author}* -- *{item.subject}*: {item.body}', config)
```
Comment (perryprog) [inline tor/core/inbox.py]: This line is too long

File: tor/core/inbox.py [hunk 5]
```diff
@@ -25,15 +25,10 @@
...
 
     logging.info(
-        'Received unhandled inbox message from {}. \nSubject: {}\n\nBody: {}'
-        ''.format(item.author, item.subject, item.body)
+        f'Received unhandled inbox message from {item.author}. \nSubject: {item.subject}\n\nBody: {item.body}'
```
Comment (perryprog) [inline tor/core/inbox.py]: This line is too long

File: tor/core/inbox.py [hunk 6]
```diff
@@ -64,10 +59,8 @@ def process_mod_intervention(post, config):
     phrases = '"' + '", "'.join(phrases) + '"'
 
     send_to_slack(
-        ':rotating_light::rotating_light: Mod Intervention Needed '
-        ':rotating_light::rotating_light: \n\nDetected use of '
-        '{phrases} <{link}>'
-        ''.format(link=post.submission.shortlink, phrases=phrases),
+        f':rotating_light::rotating_light: Mod Intervention Needed :rotating_light::rotating_light: '
```
Comment (perryprog) [inline tor/core/inbox.py]: This line is too long

File: tor/core/inbox.py [hunk 7]
```diff
@@ -140,8 +133,7 @@ def check_inbox(config):
         # In this case, there will be no author attribute.
         if item.author is None:
             send_to_slack(
-                'We received a message without an author. Subject: {}'
-                ''.format(item.subject), config
+                f'We received a message without an author. Subject: {item.subject}', config
```
Comment (perryprog) [inline tor/core/inbox.py]: This line is too long

File: tor/core/posts.py [hunk 8]
```diff
@@ -71,9 +71,7 @@ def process_post(new_post, config):
         return
 
     logging.info(
-        'Posting call for transcription on ID {} posted by {}'.format(
-            new_post.fullname, new_post.author.name
-        )
+        f'Posting call for transcription on ID {new_post.fullname} posted by {new_post.author.name}'
```
Comment (perryprog) [inline tor/core/posts.py]: This line is too long

File: tor/core/posts.py [hunk 9]
```diff
@@ -95,10 +93,7 @@ def process_post(new_post, config):
                 ))
                 add_complete_post_id(new_post.fullname, config)
                 logging.info(
-                    'Found YouTube video, {}, with good transcripts.'
-                    ''.format(
-                        get_yt_video_id(new_post.url)
-                    )
+                    f'Found YouTube video, {get_yt_video_id(new_post.url)}, with good transcripts.'
```
Comment (perryprog) [inline tor/core/posts.py]: This line is too long

File: tor/core/posts.py [hunk 10]
```diff
@@ -151,13 +146,6 @@ def process_post(new_post, config):
...
-            'ID: {id}\n'
-            'Title: {title}\n'
-            'Subreddit: {sub}'.format(
-                e,
-                id=new_post.fullname,
-                title=new_post.title,
-                sub=new_post.subreddit.display_name
-            )
+            f'{e} - unable to post content.\nID: {new_post.fullname}\n '
+            f'Title: {new_post.title}\nSubreddit: {new_post.subreddit.display_name}'
```
Comment (perryprog) [inline tor/core/posts.py]: This line is too long

File: tor/core/user_interaction.py [hunk 11]
```diff
@@ -50,10 +50,7 @@ def process_coc(post, config):
     # instead. If they're actually new, then send a message to slack.
     if result == 1:
         send_to_slack(
-            '<http://www.reddit.com/u/{user}|u/{user}> has just '
-            'accepted the CoC!'.format(
-                user=post.author.name
-            ), config
+            f'<http://www.reddit.com/u/{post.author.name}|u/{post.author.name}> has just accepted the CoC!', config
```
Comment (perryprog) [inline tor/core/user_interaction.py]: This line is too long

File: tor/core/user_interaction.py [hunk 12]
```diff
@@ -108,8 +103,7 @@ def process_claim(post, config):
     except praw.exceptions.APIException as e:
         if e.error_type == 'DELETED_COMMENT':
             logging.info(
-                'Comment attempting to claim ID {} has been deleted. '
-                'Back up for grabs!'.format(top_parent.fullname)
+                f'Comment attempting to claim ID {top_parent.fullname} has been deleted. Back up for grabs!'
```
Comment (perryprog) [inline tor/core/user_interaction.py]: This line is too long

File: tor/core/user_interaction.py [hunk 13]
```diff
@@ -144,10 +138,7 @@ def process_done(post, config, override=False):
                 # we need to double-check these things to keep people
                 # from gaming the system
                 logging.info(
-                    'Post {} does not appear to have a post by claimant {}. '
-                    'Hrm...'.format(
-                        top_parent.fullname, post.author
-                    )
+                    f'Post {top_parent.fullname} does not appear to have a post by claimant {post.author}. Hrm...'
```
Comment (perryprog) [inline tor/core/user_interaction.py]: This line is too long

File: tor/core/user_interaction.py [hunk 14]
```diff
@@ -194,8 +182,7 @@ def process_done(post, config, override=False):
     except praw.exceptions.APIException as e:
         if e.error_type == 'DELETED_COMMENT':
             logging.info(
-                'Comment attempting to mark ID {} as done has been deleted'
-                ''.format(top_parent.fullname)
+                f'Comment attempting to mark ID {top_parent.fullname} as done has been deleted'
```
Comment (perryprog) [inline tor/core/user_interaction.py]: This line is too long

**Comment by TheLonelyGhost** [comment]
There were some merge conflicts after I merged a different PR, so I resolved those on your branch. That's why it shows both of our profile pics next to each other for each commit.

**Pull request merged**